docs/conf.py

```diff
@@ -0,0 +1,113 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+
+import pathlib
+
+import toml
+
+
+# -- Project information -----------------------------------------------------
+
+project = "betterproto"
+copyright = "2019 Daniel G. Taylor"
+author = "danielgtaylor"
+pyproject = toml.load(open(pathlib.Path(__file__).parent.parent / "pyproject.toml"))
+
+
+# The full version, including alpha/beta/rc tags.
+release = pyproject["tool"]["poetry"]["version"]
+
+
+# -- General configuration ---------------------------------------------------
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = [
+    "sphinx.ext.autodoc",
+    "sphinx.ext.intersphinx",
+    "sphinx.ext.napoleon",
+]
+
+autodoc_member_order = "bysource"
+autodoc_typehints = "none"
+
+extlinks = {
+    "issue": ("https://github.com/danielgtaylor/python-betterproto/issues/%s", "GH-"),
+}
+
+# Links used for cross-referencing stuff in other documentation
+intersphinx_mapping = {
+    "py": ("https://docs.python.org/3", None),
+}
+
+# Add any paths that contain templates here, relative to this directory.
+templates_path = ["_templates"]
+
+
+# -- Options for HTML output -------------------------------------------------
+
+# The name of the Pygments (syntax highlighting) style to use.
+pygments_style = "friendly"
+
+# The theme to use for HTML and HTML Help pages.  See the documentation for
+# a list of builtin themes.
+#
+html_theme = "sphinx_iowmd_theme"
```

Comment:
Personally I rarely read docs cover to cover so I'm not a fan of having to click the burger menu to see the contents. Also I usually don't read docs on a small touch screen so a UI that optimises for that use case seems suboptimal.

Did you consider other themes?

Reply:
Also is there a way to get two levels of content in the side menu?

Reply:
Same thought~

I see that some `readthedocs` themes have a more extensive sidebar, so you can easily jump to different pages.

For example: https://docs.scrapy.org/en/latest/

To be honest, I find a lot of issues with the Material theme

- code blocks have no border or distinct color, so the page looks disorganized, and code is not readable
- the size of headings in the TOC blocks is inconsistent (higher level headings are smaller)
- no overview of pages / sidebar

I suggest we use the theme / structure of Scrapy

Reply:
I would second using the basic RTD theme for clearer presentation: https://sphinx-rtd-theme.readthedocs.io/en/stable/